<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="kaushik-Inspiron-5547" failures="0" tests="1" name="com.test.TestNGExceptionTimeOut" time="0.013" errors="1" timestamp="21 Aug 2019 06:14:51 GMT" skipped="0">
  <testcase classname="com.test.TestNGExceptionTimeOut" name="myTest" time="0.013">
    <error message="Method com.test.TestNGExceptionTimeOut.myTest() didn&#039;t finish within the time-out 1" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.test.TestNGExceptionTimeOut.myTest() didn't finish within the time-out 1
at java.base@11.0.4/java.io.FileOutputStream.writeBytes(Native Method)
at java.base@11.0.4/java.io.FileOutputStream.write(FileOutputStream.java:354)
at java.base@11.0.4/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
at java.base@11.0.4/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:142)
at java.base@11.0.4/java.io.PrintStream.write(PrintStream.java:561)
at java.base@11.0.4/sun.nio.cs.StreamEncoder.writeBytes(StreamEncoder.java:233)
at java.base@11.0.4/sun.nio.cs.StreamEncoder.implFlushBuffer(StreamEncoder.java:312)
at java.base@11.0.4/sun.nio.cs.StreamEncoder.flushBuffer(StreamEncoder.java:104)
at java.base@11.0.4/java.io.OutputStreamWriter.flushBuffer(OutputStreamWriter.java:184)
at java.base@11.0.4/java.io.PrintStream.newLine(PrintStream.java:625)
at java.base@11.0.4/java.io.PrintStream.println(PrintStream.java:883)
at app//com.test.TestNGExceptionTimeOut.myTest(TestNGExceptionTimeOut.java:14)
at java.base@11.0.4/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@11.0.4/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base@11.0.4/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base@11.0.4/java.lang.reflect.Method.invoke(Method.java:566)
at app//org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at app//org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at app//org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:40)
at java.base@11.0.4/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
at java.base@11.0.4/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@11.0.4/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base@11.0.4/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base@11.0.4/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- myTest -->
</testsuite> <!-- com.test.TestNGExceptionTimeOut -->
